doc/templates/index.html

```diff
@@ -186,11 +186,7 @@ <h4 class="sk-landing-call-header">Community</h4>
         <li>Communication on all channels should respect <a href="https://www.python.org/psf/conduct/">PSF's code of conduct.</a></li>
         </ul>
 
-        <form target="_top" id="paypal-form" method="post" action="https://www.paypal.com/cgi-bin/webscr">
-          <input type="hidden" value="_s-xclick" name="cmd">
-          <input type="hidden" value="74EYUMF3FTSW8" name="hosted_button_id">
-        </form>
-        <a class="btn btn-warning btn-big sk-donate-btn mb-1" onclick="document.getElementById('paypal-form').submit(); ">Help us, <strong>donate!</strong></a>
+        <a class="btn btn-warning btn-big sk-donate-btn mb-1" href="https://numfocus.salsalabs.org/supportscikitlearn">Help us, <strong>donate!</strong></a>
```

Comment:
Cf concerns about this domain name in https://github.com/scikit-learn/scikit-learn/issues/18345#issuecomment-687574129